monai/bundle/scripts.py

```diff
@@ -712,11 +734,11 @@ def run(
     config_file_, meta_file_, init_id_, run_id_, final_id_, logging_file_, tracking_ = _pop_args(
         _args,
         config_file=None,
-        meta_file="configs/metadata.json",
+        meta_file=None,
         init_id="initialize",
         run_id="run",
         final_id="finalize",
-        logging_file="configs/logging.conf",
+        logging_file=None,
         tracking=None,
     )
     workflow = ConfigWorkflow(
```

Comment:
I guess this could now be implemented with `create_workflow`?

Reply:
Yes, have updated it. Thanks!